packages/grid/x-data-grid/src/tests/columnsVisibility.DataGrid.test.tsx

```diff
@@ -78,7 +75,6 @@ describe('<DataGridPro /> - Columns Visibility', () => {
       expect(onColumnVisibilityModelChange.callCount).to.equal(1);
       expect(onColumnVisibilityModelChange.lastCall.firstArg).to.deep.equal({
         id: false,
-        idBis: true,
```

Comment:
Should we warn dev about the fact that ?

```js
// Good
isVisible = columnVisibilityModel[field] !== false
// Good
isVisible = columnVisibilityModel[field] ?? true
// Wrong
isVisible = columnVisibilityModel[field]
```

Reply:
I think once we had done #6037 we can add a small note in the doc :+1: